Source/WebKit/WebProcess/cocoa/VideoFullscreenManager.h

```diff
@@ -119,7 +117,7 @@ class VideoFullscreenInterfaceContext
     RetainPtr<CALayer> m_rootLayer;
 };
 
-class VideoFullscreenManager : public RefCounted<VideoFullscreenManager>, private IPC::MessageReceiver {
+class VideoFullscreenManager : public RefCounted<VideoFullscreenManager>, public IPC::MessageReceiver {
```

Comment:
I'd have rather avoided making this `public`, but I'm not quite sure how else to make the conformance to `CanMakeWeakPtr` visible.